core/src/test/scala/com.codacy.analysis.core/git/RepositorySpec.scala

```diff
@@ -35,5 +37,68 @@ class RepositorySpec extends Specification with NoLanguageFeatures {
       }
     }
 
+    "get all uncommited changes" in {
+
+      "changed files" in {
+        withTemporaryGitRepo { directory =>
+          (for {
+            file <- File.temporaryFile(parent = Some(directory))
+          } yield {
+
+            Process(Seq("git", "add", "."), directory.toJava).!
+            Process(Seq("git", "commit", "-m", "geez rick! i-i-i-i-i a-a-a-added a file!"), directory.toJava).!
+
+            file.write("Wubba lubba dub dub!")
+
+            Git.repository(directory).flatMap(_.uncommitedFiles) must beLike {
+              case Success(uncommited) =>
+                uncommited.foreach(println)
+                uncommited must contain(relativePath(file, directory))
```

Comment:
This should be a Seq comparison instead of a `must contain`, because it must not contain anything else.

Do this for the other tests as well.

Reply:
or even better:
```scala
uncommited must contain(exactly(relativePath(file, directory)))
```